### DISCUSSION TITLE
API: Store file in db then attach url to attachment field [Mar 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5282] [Discord Threads]
> 🙋‍♂️ **Rahim Dobani**

**Comment by o1lab** [comment]
> 🙋‍♂️ **kyleecodes**  
#### Tagging <@&682859083799199814>  
#  
> 👨‍💻 **pranavxc**  
#### Have a look at the following discussion and our API documentation.   
####    
#### https://github.com/nocodb/nocodb/discussions/369   
#### https://all-apis.nocodb.com/#tag/Storage/operation/storage-upload  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1079700975528452106)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
